### PR TITLE
Edit operation

### PR DESCRIPTION
**Pull request by telepenin**
- apiserver
- CLI
- add error message for 404 status
- not required for operator's replicas field

### CODE REVIEW

File: modules/dynamic-apiserver/internal/app/service_edit.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/internal/generated/models"
+	apiService "github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/internal/generated/restapi/operations/service"
+	"github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/internal/util"
+	kuberlogiccomv1alpha1 "github.com/kuberlogic/kuberlogic/modules/dynamic-operator/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// set this string to a required security grant for this action
+const serviceEditSecGrant = "nonsense"
```
Comment (ynnt) [inline modules/dynamic-apiserver/internal/app/service_edit.go:15]: is it still needed?

File: modules/dynamic-apiserver/internal/app/service_get.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	"fmt"
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/internal/generated/models"
+	apiService "github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/internal/generated/restapi/operations/service"
+	"github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/internal/util"
+	kuberlogiccomv1alpha1 "github.com/kuberlogic/kuberlogic/modules/dynamic-operator/api/v1alpha1"
+)
+
+// set this string to a required security grant for this action
+const serviceGetSecGrant = "nonsense"
```
Comment (ynnt) [inline modules/dynamic-apiserver/internal/app/service_get.go:13]: same as above

File: modules/dynamic-apiserver/openapi.yaml [hunk 3]
```diff
@@ -88,6 +88,81 @@ paths:
...
+        401:
+          description: bad authentication
+        403:
+          description: bad permissions
+        404:
+          description: item not found
+          schema:
+            $ref: "#/definitions/Error"
+
+        409:
```
Comment (ynnt) [inline modules/dynamic-apiserver/openapi.yaml:156]: We do not expect to create a service item here

File: modules/dynamic-apiserver/openapi.yaml [hunk 4]
```diff
@@ -88,6 +88,81 @@ paths:
...
+        401:
+          description: bad authentication
+        403:
+          description: bad permissions
+        404:
+          description: item not found
+          schema:
+            $ref: "#/definitions/Error"
+        409:
+          description: item already exists
```
[the remark this answers is not in the document]
Comment (telepenin) [reply modules/dynamic-apiserver/openapi.yaml:118]: Agree

File: modules/dynamic-apiserver/openapi.yaml (hunk 3 above)
Comment (telepenin) [reply modules/dynamic-apiserver/openapi.yaml:156]: Agree

File: modules/dynamic-apiserver/internal/app/service_edit.go (hunk 1 above)
Comment (telepenin) [reply modules/dynamic-apiserver/internal/app/service_edit.go:15]: No, I'll remove it.

File: modules/dynamic-apiserver/internal/app/service_get.go (hunk 2 above)
Comment (telepenin) [reply modules/dynamic-apiserver/internal/app/service_get.go:13]: Thanks

**Pull request merged**